gcp_variant_transforms/libs/variant_sharding.py

```diff
@@ -78,13 +74,13 @@ def add_region(self, start, end, shard_index):
       raise ValueError(
           'Index of a region cannot be negative {}'.format(shard_index))
     if self._interval_tree.overlaps_range(start, end):
-      raise ValueError(
-          'Cannot add overlapping region {}-{}'.format(start, end))
+      raise ValueError('Wrong sharding config file, regions must be unique in '
+                       'config file: {}-{}'.format(start, end))
     # If everything goes well we add the new region to the interval tree.
     self._interval_tree.addi(start, end, shard_index)
 
   def get_shard_index(self, pos=0):
-    """Finds a region that includes pos, if none _UNDEFINED_PARTITION_INDEX."""
+    """Finds a region that includes pos, if none _UNDEFINED_SHARD_INDEX."""
```

Comment:
nit: I know it was wrong in the original version, but replace with "..., if none _return_ _UNDEFINED_SHARD_INDEX."

Reply:
Sorry, missed this from last round.
Done.